src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -987,3 +987,658 @@
+    type: int
+    short-summary: >
+        The port used for http requests to the origin.  Default: 80.
+  - name: --https-port
+    type: int
+    short-summary: >
+        The port used for https requests to the origin.  Default: 443.
+  - name: --origin-host-header
+    type: string
+    short-summary: >
+        The Host header to send for requests to this origin. If you leave this blank, the request hostname determines this value.
+        Azure CDN origins, such as Web Apps, Blob Storage, and Cloud Services require this host header value to match the origin hostname by default.
+  - name: --origin-group-name
+    type: string
+    short-summary: >
+        The Name of the origin group.
+  - name: --weight
+    type: int
+    short-summary: >
+        The weight of the origin in given origin group for load balancing. Must be between 1 and 1000.
+  - name: --priority
+    type: int
+    short-summary: >
+        The load balancing priority. Higher priorities will not be used for load balancing if any lower priority origin is healthy.
+        Must be between 1 and 5.
+  - name: --enabled-state
+    type: bool
+    short-summary: >
+        Whether to enable this origin. Allowed values: Disabled, Enabled.
+  - name: --origin-name
+    type: string
+    short-summary: >
+        Name of the origin.
+  - name: --private-link-resource
+    type: string
+    short-summary: >
+        The resource ID of the origin that will be connected to using the private link.
+  - name: --private-link-location
+    type: string
+    short-summary: >
+        The location of origin that will be connected to using the private link.
+  - name: --private-link-request-message
+    type: string
+    short-summary: >
+        The message that is shown to the approver of the private link request.
+  - name: --private-link-sub-resource-type
+    type: string
+    short-summary: >
+        The sub-resource type of the origin that will be connected to using the private link.
+        You can use "az network private-link-resource list" to obtain the supported sub-resource types.
+examples:
+  - name: Update the host header and priority of the specified origin.
+    text: >
+      az afd origin update -g group --host-name example.contoso.com --profile-name profile --origin-group-name originGroup
+      --origin-name origin1 --origin-host-header example.contoso.com --priority 3
+  - name: Disable private link of the origin.
+    text: >
+      az afd origin update -g group --host-name example.contoso.com --profile-name profile --origin-group-name originGroup
+      --origin-name origin1 --enable-private-link False
+"""
+
+helps['afd custom-domain'] = """
+type: group
+short-summary: Manage custom domains within the specified profile.
+"""
+
+helps['afd custom-domain create'] = """
+type: command
+short-summary: Create a custom domain within the specified profile. Your need to create a DNS TXT record to prove the custom domain's ownership to make the operation successful.
```

Comment:
This summary is too long. You should move some part of summary into `long-summary`